code/client/src/util.js

```diff
@@ -4,6 +4,7 @@ import { isInteger, values } from 'lodash'
 import ONEUtil from '../../lib/util'
 import { AddressError } from './constants/errors'
 import config from './config'
+import api from './api'
```

Comment:
Let's minimize the dependencies in util since many components may depend on this, including api itself